src/saiga/vision/util/Features.h

```diff
@@ -6,32 +6,90 @@
 
 #pragma once
 
+#include <opencv2/core/types.hpp>
```

Comment:
Remove this dependency. Saiga features should work without opencv. (That's why the keypoint is copied in the first place)